index.html

```diff
@@ -25,15 +26,174 @@ <h1> I’m George
+                </ul>
+            </div>
+
+            <a href="#"> <div class="btn-main">See Project</div> </a> 
+        </section>
+    </section>
+
+    <section class="snapshot1">
+        <img src="img/Snapshoot-Portfolio3.png" alt="snapshot1" class="image-po">
+        <section id="main">            
+            <div class="head-one">
+                <h1>Tonic</h1>
+            </div>
+                <div class="grid-container">
+                    <div class="grid-item"> <span class="canop">CANOPY</span></div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> Back End Dev</div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2015</div>  
+                </div>         
+            
+            <div class="doc">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.                    
+                </p>
+            </div>
+            <div class="tags">
+                <ul class="works">
+                    <li> <a href="#" class="btn"> html </a></li>
+                    <li> <a href="#" class="btn"> css </a></li>
+                    <li> <a href="#" class="btn"> javascript </a></li>
+                </ul>
+            </div>
+            <a href="#"> <div class="btn-main">See Project</div> </a> 
+            
+        </section>
+    </section>
+
+    <section class="snapshot1">
+        <img src="img/Snapshoot Portfolio4.png" alt="snapshot1" class="image-po">
+        <section id="main">            
+            <div class="head-one-alt">
+                <h1>Multi-Post
+                    Stories</h1>
+            </div>
+                <div class="grid-container">
+                    <div class="grid-item"> <span class="canop">CANOPY</span></div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> Back End Dev</div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2015</div>  
+                </div>               
+            
+            <div class="doc">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.                    
+                </p>
+            </div>
+            <div class="tags">
+                <ul class="works">
+                    <li> <a href="#" class="btn"> html </a></li>
+                    <li> <a href="#" class="btn"> css </a></li>
+                    <li> <a href="#" class="btn"> javascript </a></li>
+                </ul>
+            </div>
+            <a href="#"> <div class="btn-main">See Project</div> </a> 
+            
+        </section>
+    </section>
+
+    <section class="about">
```

Comment:
- [ ] Kindly ensure that you use the correct margins right and left in your about me section to match the chosen Figma design

**This is your portfolio right now**

![image](https://user-images.githubusercontent.com/86318284/193089459-555fa9b4-311e-4043-83c4-3c2dff97fd73.png)

**Thi is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/193089552-0350edf7-cbf7-4d45-ab11-5a7c7c8890a1.png)